test/test_torch.py

```diff
@@ -15936,10 +15936,10 @@ def transformation_cuda_fn(tensor, **kwargs):
             self._test_memory_format_transformations(
                 'cpu', get_generator(mf, shape), transformation_cuda_fn, mf, default_is_preserve=True)
 
-    @onlyCPU
     @skipCPUIfNoLapack
+    @skipCUDAIfNoMagma
     @dtypes(torch.double)
-    def test_eig(self, device, dtype):
+    def test_eig_basic(self, device, dtype):
```

Comment:
Let's move the eig tests to test_linalg.py

Reply:
done in 17e3d511a5